File: BlogApi/BlogApi/Models/Post.cs
using System;
using System.Collections.Generic;
//////////////////////////////
// Poglavlje 2.3.1 - Kreiranje modela
/////////////////////////////
namespace BlogApi.Models
{
    public class Post
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Content { get; set; }
        public ICollection<Comment> Comments { get; set; }
        public User CreatedBy { get; set; }
        public Guid CreatedById { get; set; }
        public DateTime CreatedOn { get; set; }
        public DateTime ModifiedOn { get; set; }
    }
}

File: BlogSpa/src/app/_models/post/post.ts
// Predstavlja model za novi post
// Potencijalno prosirenje u buducnost bilo bi generisanje modela iz serverske strane
// ? oznacava property nije obavezan
export interface Post {
  id?: string;
  name: string;
  content: string;
  createdById?: string;
  createdBy?: any;
  createdOn?: Date;
  modifiedOn?: Date;
}
